src/django/pfb_analysis/models.py

```diff
@@ -348,17 +361,53 @@ def create_environment(**kwargs):
         container_overrides = {
             'environment': create_environment(**environment),
         }
-        response = client.submit_job(jobName=self.batch_job_name,
-                                     jobDefinition=settings.PFB_AWS_BATCH_JOB_DEFINITION_NAME_REVISION, # NOQA
-                                     jobQueue=settings.PFB_AWS_BATCH_JOB_QUEUE_NAME,
-                                     containerOverrides=container_overrides)
+        response = client.submit_job(
+            jobName=self.analysis_job_name,
+            jobDefinition=settings.PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME_REVISION,
+            jobQueue=settings.PFB_AWS_BATCH_ANALYSIS_JOB_QUEUE_NAME,
+            containerOverrides=container_overrides)
         try:
             self.batch_job_id = response['jobId']
             self.save()
             self.update_status(self.status.QUEUED)
         except (botocore.exceptions.BotoCoreError, KeyError):
             logger.exception('Error starting AnalysisJob {}'.format(self.uuid))
 
+    def generate_tiles(self):
```

Comment:
I made EXPORTED and TILING statuses, the former being the new final status of the analysis container.  The tiling container sets TILING when it starts and COMPLETE when it's done.